migrations/15.0.1.4.0/pre-migration.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 Tecnativa - Carolina Fernandez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(env):
+    env.cr.execute(
+        """
+        ALTER TABLE resource_booking RENAME partner_id TO migrated_partner_id
```

Comment:
Better to use a familiar nomenclature (and more expressive):

```suggestion
        ALTER TABLE resource_booking RENAME partner_id TO old_partner_id
```

Reply:
And technically FYI, there's no need to rename the column, as this column is available on post-migration.

Reply:
Ok,  so if the column is available i delete the file pre-migration.py

Reply:
Oh, wait, I didn't see that you are still defining `partner_id` as a regular field, but not stored, so in this case the column is removed, so please rename. Sorry for not seeing before that issue.